tests/scanner/audit/data/bucket_retention_test_rules_1.yaml

```diff
@@ -0,0 +1,8 @@
+rules:
```

Comment:
nit: i am not a fan of test cases spread amongst files. consider using the test style from https://github.com/GoogleCloudPlatform/forseti-security/pull/2075 (once approved).

Reply:
From test_rules_0 to test_rules_13, each of them rises an error. That is the reason why I use different files. test_rules_2* covers all the other logic.